flight/Modules/Attitude/attitude.c

```diff
@@ -1410,7 +1406,6 @@ static void updateNedAccel()
 	accel_ned[2] += GRAVITY;
 	
 	NedAccelData accelData;
-	NedAccelGet(&accelData);
```

Comment:
This one is set by GroundPathFollower as well, https://github.com/d-ronin/dRonin/blob/080150a5b271fcfb3fe36aa1e20f92fc0d956aca/flight/Modules/GroundPathFollower/groundpathfollower.c#L484